lib/toolbar/view-toolbar.vue

```diff
@@ -139,3 +105,42 @@
     }
   };
 </script>
+
+<style lang="sass">
+  @import '../../styleguide/toolbar';
+  @import '../../styleguide/layers';
+
+  .kiln-wrapper.view-mode {
+    @include toolbar-wrapper();
+
+    pointer-events: none;
+
+    & > * {
+      pointer-events: all;
+    }
+
+    .view-menu-button {
+      margin: 10px;
+    }
+
+    .kiln-progress {
+      height: 3px;
+      left: 0;
+      position: fixed;
+      top: 0;
+      width: 100%;
+    }
+
+    .view-edit-button,
+    .view-new-button {
+      margin: 10px 18px;
+    }
+
+    .ui-snackbar-container {
+      @include confirm-layer();
+
+      bottom: 0;
+      position: fixed;
+    }
+  }
+</style>
```

Comment:
Why did we move the styles to the bottom of the file? 😛

Reply:
Because that's the standard Vue order of tags. Template, Script, then Styles.

Reply:
From the [vuejs.org style guide](https://vuejs.org/v2/style-guide/) 
"Single-file components should always order `<script>`, `<template>`, and `<style>` tags consistently, with `<style>` last, because at least one of the other two is always necessary."